modules/gmc-agent/src/main/java/org/gridgain/dto/cluster/BaselineParameters.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gridgain.dto.cluster;
+
+import org.apache.ignite.internal.util.typedef.internal.S;
+
+import java.util.Objects;
+
+/**
+ * DTO for baseline parameters.
+ */
+public class BaselineParameters {
+    /** Is auto adjust enabled. */
+    private boolean isAutoAdjustEnabled;
+
+    /** Auto adjust awaiting time. */
+    private long autoAdjustAwaitingTime;
+
+    /**
+     * Default constructor.
+     */
+    public BaselineParameters() {
+        // No-op.
+    }
+
+    /**
+     * @param isAutoAdjustEnabled Is auto adjust enabled.
+     * @param autoAdjustAwaitingTime Auto adjust awaiting time.
+     */
+    public BaselineParameters(boolean isAutoAdjustEnabled, long autoAdjustAwaitingTime) {
+        this.isAutoAdjustEnabled = isAutoAdjustEnabled;
+        this.autoAdjustAwaitingTime = autoAdjustAwaitingTime;
+    }
+
+    /**
+     * @return @{code true} if auto adjust is enabled.
+     */
+    public boolean isAutoAdjustEnabled() {
+        return isAutoAdjustEnabled;
+    }
+
+    /**
+     * @param autoAdjustEnabled Auto adjust enabled.
+     * @return @{code This} for method chaining.
+     */
+    public BaselineParameters setAutoAdjustEnabled(boolean autoAdjustEnabled) {
+        isAutoAdjustEnabled = autoAdjustEnabled;
+        return this;
+    }
+
+    /**
+     * @return Auto adjust awaiting time in ms.
+     */
+    public long getAutoAdjustAwaitingTime() {
+        return autoAdjustAwaitingTime;
+    }
+
+    /**
+     * @param autoAdjustAwaitingTime Auto adjust awaiting time.
+     * @return @{code This} for method chaining.
+     */
+    public BaselineParameters setAutoAdjustAwaitingTime(long autoAdjustAwaitingTime) {
+        this.autoAdjustAwaitingTime = autoAdjustAwaitingTime;
+        return this;
+    }
+
+    /** {@inheritDoc} */
+    @Override public String toString() {
```

Comment:
Move this method the end of file.
Because this method is the last one for POJOs.

Reply:
Done.